datahub-frontend/conf/application.conf

```diff
@@ -139,6 +139,7 @@ auth.oidc.responseMode = ${?AUTH_OIDC_RESPONSE_MODE}
 auth.oidc.useNonce = ${?AUTH_OIDC_USE_NONCE}
 auth.oidc.customParam.resource = ${?AUTH_OIDC_CUSTOM_PARAM_RESOURCE}
 auth.oidc.readTimeout = ${?AUTH_OIDC_READ_TIMEOUT}
+auth.oidc.parseJwtAccessTokenEnabled = ${?AUTH_OIDC_PARSE_JWT_ACCESS_TOKEN_ENABLED} # Whether to parse claims from JWT access token.  Defaults to false.
```

Comment:
Awesome!

Reply:
minor: we can consider an alternate name here as well:

AUTH_OIDC_EXTRACT_JWT_ACCESS_TOKEN_CLAIMS

But i have no strong preference - up to you!